lib/models/store_transaction.dart

```diff
@@ -11,13 +11,15 @@ class StoreTransaction with _$StoreTransaction {
   /// Experimental. This factory method is subject to changes without
   /// a major release.
   const factory StoreTransaction.create(
-    /// RevenueCat Id associated to the transaction.
+    /// RevenueCat Id associated to the transaction. Empty for Amazon.
```

Comment:
Maybe this to be more specific? Since it will not be empty for Amazon inapps I believe.
```suggestion
    /// RevenueCat Id associated to the transaction. Empty for Amazon subscriptions.
```
I believe this will only be empty there but lmk if I'm wrong.

Reply:
Ohh ok... I was thinking we would be using the `Transaction` model for inapps but maybe that's not correct for paywalls... I think this is ok then